src/links.js

```diff
@@ -15,17 +15,21 @@ const links = {
 
     landing_worker_compensation: '/worker-compensation',
 
+    landing_blog: '/blog',
+
     landing_whitepaper:
         'https://github.com/gnosis/dx-daostack/blob/master/dxdao_whitepaper_v1.pdf',
 
     landing_mixeth:
         'https://daotalk.org/t/mix-eth-seeking-feedback-on-proposal/1183',
 
-    landing_omeneth: 'https://omen.eth.link',
+    landing_omeneth: 'https://github.com/protofire/omen-exchange/projects/1',
```

Comment:
Since we're linking to the codebase instead of the landing page now, we should change the variable to something like `codebase_omeneth`. Same for the other variables below that have had this change.